pkg/controller/flow/reconcile.go

```diff
@@ -266,26 +261,22 @@ func (r *reconciler) createSubscription(channelName string, target string, flow
 }
 
 func (r *reconciler) reconcileFeed(channelDNS string, flow *v1alpha1.Flow) (*feedsv1alpha1.Feed, error) {
-	feedName := flow.Name
-
-	feed := &feedsv1alpha1.Feed{}
-	err := r.client.Get(context.TODO(), client.ObjectKey{Namespace: flow.Namespace, Name: feedName}, feed)
+	feed, err := r.getFeed(flow)
 	if errors.IsNotFound(err) {
 		feed, err = r.createFeed(channelDNS, flow)
 		if err != nil {
-			glog.Errorf("Failed to create feed %q : %v", feedName, err)
+			glog.Errorf("Failed to create feed for flow %q: %v", flow.Name, err)
```

Comment:
For a subsequent PR, but if you convert this to Zap, you can put the flow name in the logger's context at the start of the function and not need to include it on each log message.

Reply:
Ack.